File: projects/account-manager/dotnet/backend/Models/Account.cs
namespace AccountManager.Models;

public class Account
{
    public long Id { get; set; }
    public string Name { get; set; } = "";
    public string? Address { get; set; }
    public string Email { get; set; } = "";
    public int Status { get; set; } = 1;
    public long CreatedTime { get; set; }
    public long UpdatedTime { get; set; }
}

File: projects/account-manager/java/backend/src/main/java/com/example/accountmanager/model/Account.java
package com.example.accountmanager.model;

public class Account {
    private Long id;
    private String name;
    private String address;
    private String email;
    private int status;
    private long createdTime;
    private long updatedTime;

    public Account() {}

    public Long getId() { return id; }
    public void setId(Long id) { this.id = id; }

    public String getName() { return name; }
    public void setName(String name) { this.name = name; }

    public String getAddress() { return address; }
    public void setAddress(String address) { this.address = address; }

    public String getEmail() { return email; }
    public void setEmail(String email) { this.email = email; }

    public int getStatus() { return status; }
    public void setStatus(int status) { this.status = status; }

    public long getCreatedTime() { return createdTime; }
    public void setCreatedTime(long createdTime) { this.createdTime = createdTime; }

    public long getUpdatedTime() { return updatedTime; }
    public void setUpdatedTime(long updatedTime) { this.updatedTime = updatedTime; }
}
